flight/Modules/Autotune/autotune.c

```diff
@@ -361,7 +367,9 @@ static void AutotuneTask(void *parameters)
 		PIOS_Thread_Sleep(25);
 	}
 
-	uint16_t buf_size = sizeof(*at_averages) * ident_wiggle_points;
+	decim_wiggle_points /= AUTOTUNE_AVERAGING_DECIMATION;
```

Comment:
I don't see this being initialized anywhere? Is it supposed to be `decim_wiggle_points = ident_wiggle_points / AUTOTUNE_AVERAGING_DECIMATION`?